spec/classes/puppet_agent_osfamily_aix_spec.rb

```diff
@@ -13,7 +13,7 @@
   end
 
   package_version = '1.2.5'
-  package_ensure = 'present'
+  package_ensure = Puppet.version < "4.0.0" ? 'present' : '1.2.5'
```

Comment:
Couldn't this '1.2.5' be set to `package_version`?

Reply:
Yes, done.